tests/integration/returners/local_cache.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+'''
+Tests for the local_cache returner
+'''
+# Import Python libs
+from __future__ import absolute_import
+import logging
+import os
+import platform
+import tempfile
+
+# Import Salt Testing libs
+from salttesting.helpers import ensure_in_syspath
+ensure_in_syspath('../../')
+
+# Import salt libs
+import integration
+import salt.utils.job
+from salt.returners import local_cache
+
+log = logging.getLogger(__name__)
+if platform.uname()[0] == 'Darwin':
```

Comment:
Much this logic is available from `salt.integration` already. i.e. `salt.integration.TMP`.